wordle-web/src/views/WordleView.vue

```diff
@@ -17,19 +52,100 @@ import { ref, reactive, onMounted, onUnmounted } from 'vue'
 import GameBoard from '../components/GameBoard.vue'
 import KeyBoard from '../components/KeyBoard.vue'
 import type { Letter } from '@/scripts/letter'
+import Axios from 'axios'
+
+import UsernameDialog from '../components/UsernameDialog.vue'
+import { PlayerService } from '@/scripts/playerService'
+import { Player } from '@/scripts/player'
 
 const guess = ref('')
 const game = reactive(new WordleGame())
 
-onMounted(async () => {
-  window.addEventListener('keyup', keyPress)
-  await game.restartGame()
-  console.log(game.secretWord)
+const overlay = ref(true)
+const newPlayer = ref(true)
+const currentPlayerName = ref('')
+
+let user = new Player(0, 'Guest', 0, 0, 0)
```

Comment:
I'm not seeing where if the player intentionally leaves the name field blank, if you then re-default their name to 'Guest'

Reply:
Good catch! code has been updated :)